src/push/request.ts

```diff
@@ -58,8 +58,14 @@ export async function sendReqAndHandleError<T>(
   options: APIRequestOptions
 ): Promise<T> {
   const { statusCode, body } = await sendRequest(options);
+
   return (
-    await handleError(statusCode, options.url, body)
+    await handleError(
+      statusCode,
+      options.url,
+      body,
+      `${options.body?.length} number of monitors were sent, with a total size of ${options.body?.length} bytes`
```

Comment:
Feels weird, Why not Kibana API return proper error message when we go over the limits?

Reply:
kibana doesn't return the payload size

Reply:
In this case, we need to do this only for the 413 status code, Otherwise we are showing this error message for all the error codes which feels incorrect.

Reply:
done